tests/common.go

```diff
@@ -2890,11 +2891,19 @@ func CreateCloudCredential(provider, name string, uid, orgID string) {
 					},
 				},
 			}
-			//ctx, err := backup.GetPxCentralAdminCtx()
-			ctx, err := backup.GetAdminCtxFromSecret()
-			expect(err).NotTo(haveOccurred(),
-				fmt.Sprintf("Failed to fetch px-central-admin ctx: [%v]",
-					err))
+			var ctx context1.Context
+			var err error
+			if UserContextBackupLocation == "" {
```

Comment:
Few points here:

- Naming the variable`UserContextBackupLocation` may not be appropriate because you are just storing the userName. You can name it something like `nonAdminUserInFocus`
- Do not hard code the password here. It is OK to hard code it in the steps of the test case but this method maybe used by users whose password is not `Password1`